### PR TITLE
Update UG - Expired command and duplicate food items

### LINKED ISSUE #126: Update UG - Expired command and duplicate food items

### PR DESCRIPTION
**Pull request by jonahtanjz**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -155,6 +157,7 @@ Format: `edit INDEX [d/DESCRIPTION] [p/PRIORITY] [q/QUANTITY] [e/EXPIRY DATE] [t
 * Existing values will be updated to the input values.
 * When editing tags, the existing tags of the food item will be removed i.e adding of tags is not cumulative.
 * You can remove all the tags of a food item by typing `t/` without specifying any tags after it.
+* Similar to [`add`](#adding-a-food-item-add) command, a food item with the same description, expriy date and tags as another food item is considered a duplicate.
```
Comment (bchenghi) [inline docs/UserGuide.md]: Should the link be #adding-a-food-item--add instead?

**Review by bchenghi — COMMENTED** [review]
The changes look good! I added one comment below.

File: docs/UserGuide.md (hunk 1 above)
Comment (jonahtanjz) [reply docs/UserGuide.md]: Alright updated. Thanks

File: docs/UserGuide.md [hunk 2]
```diff
@@ -78,7 +78,9 @@ Adds a food item to the food inventory.
 
 Format: `add d/DESCRIPTION e/EXPIRY_DATE q/QUANTITY [p/PRIORITY] [t/TAG]…​`
 
-* Adds a food item based on its description and expiry date.
+* Adds a food item based on its description, expiry date and tags.
+* A food item with the same description, expriy date and tags as another food item is considered a duplicate.
```
Comment (rolandyuwy) [inline docs/UserGuide.md]: Small typo for 'expiry', same for other lines in the UG

**Review by rolandyuwy — COMMENTED** [review]
Small typo, otherwise LGTM!

File: docs/UserGuide.md (hunk 2 above)
Comment (jonahtanjz) [reply docs/UserGuide.md]: Alright updated. Thanks.

**Review by bchenghi — APPROVED** [review]
LGTM!

**Pull request merged**